aws/util.sh

```diff
@@ -5,7 +5,40 @@
 # All providers are expected to support the following functions:
 #	util::get_instance_info
 #	util::copy_file
```

Comment:
add "util::remote_cmd" to list -- my bad

Reply:
Will add